data-de/event_scripts.s

```diff
@@ -859,7 +859,7 @@ Common_EventScript_NoRoomForDecor:: @ 81A02B8
 	msgbox Text_NoRoomLeftForAnother, MSGBOX_DEFAULT
 	return
 
-Common_EventScript_SetLegendaryWeather:: @ 81A02C1
+Common_EventScript_SetAbnormalWeather:: @ 81A02C1
```

Comment:
The old name is fine here, the `Abnormal` from pokeemerald refers to `WEATHER_ABNORMAL`, which is an Emerald-exclusive weather that alternates between drought and heavy rain.